README.md

```diff
@@ -32,6 +32,23 @@ docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/mem
 
 Memos should be running at [http://localhost:5230](http://localhost:5230). If the `~/.memos/` does not have a `memos_prod.db` file, then memos will auto generate it.
 
+## Deploy with Docker Compose
+
+The advantages of using Docker Compose to deploy are that you don't have to remember the start up command or the relationship of the containers, and it's simple to migrate all of your containers' data. 
```

Comment:
I think README should catch the point about how to run with `docker-compose`. So I'd like to write the full description in the future docs.